### ISSUE TITLE
Default title resource manager

### ISSUE THREAD

**Issue by Tychodewaard** [dnnsoftware/Dnn.Platform#4610]
## Description of problem
The resource manager has the option to create a title for a file (good!). But it is by default empty and by default WCAG (accessibillity) prescribes that all images have a title / alt-tag. 

## Description of solution
Make the default value of a file the same as the name. 

![image](https://user-images.githubusercontent.com/9091997/114873799-f3f1e300-9dfb-11eb-80dd-8385c48ddfe1.png)

**Comment by EPTamminga** [comment]
Good idea. Additional suggestions:
* Do not use the file extension in the title
* Use a standard character replacement, eg. change `"_"` into `" "`

**Comment by valadas** [comment]
Hmm, I never even noticed that, but where is it exposed after that, is it part of the DNN FileInfo? how would someone leverage that information into let's say an html image tag ?

**Comment by sleupold** [comment]
I am strictly against copying the tile into name field, even in a modified feature
- any redundancy of values should be avoided, unless there is a valid reason
- if the file gets renamed. the dn file name would not be adjusted accordingly, which might lead into confusion

the field should be left blank and a rule established, how the DNN name is generated from file name, if empty.

**Comment by mitchelsellers** [comment]
Although WCAG requires a title, a title that is the same as the filename does not actually meet the standard, additionally, I do not believe that this would/could be used for the ALT text.  I don't believe we should default this value

**Comment by sleupold** [comment]
According to W3C about name property of an img tag:
"This attribute names the element so that it may be referred to from style sheets or scripts. Note. This attribute has been included for backwards compatibility. "
For accessibility, WCAG requires a short descriptive ALT text, and I agree with @mitchelsellers , this shouldn't generated automatically from file name.

**Comment by sleupold** [comment]
IMHO, file properties "title" and "description" are great for file lists, e.g. downloads etc. but should not be used as attributes for HTML tags like image alt. it would be great, to get title and description for folders as well, which could be used in modules (via templates, like e.g. in Evotiva user files)

**Comment by Tychodewaard** [comment]
@mitchelsellers @sleupold I understand your concerns, so let's see if we can address the underlying issue:

- Because there is a title field, it looks like it is for accessibility purpose. If it is not and it is just a field that is present by default, we should consider removing it to prevent confusion
- I am always for the lookout on WCAG issues as it is mandatory in The Netherlands (probably in the entire EU) . Since 2020 if the CMS does not met the requirements, it is not used. This rule is applicable for all government, local county and a lot of non profit websites. 
- If there is a way to help editors meet the non-technical requirements, it would be most welcome.

**Comment by sleupold** [comment]
@Tychodewaard 
The title field was not intended as a SEO feature, to be used as image title when placing an image in content.
It is very helpful in a number of use cases, e.g. when providing a list of documents for download with a user friendly name and description.
